1) Line 9 prints `values added: 20`
2) Line 13 will print `final result: 20`
3) Line 9 will also print `values added: 20`
4) However in this case line 13 will return an error because the variable `result` is defined inside the if block using `let`, which means it's not accessible outside of that block's scope.
5) Line 9 in this block of code would result in an error because a const variable can not be reassigned, so when you run line 7 the code would stop.
6) For the same reason as above, line 13 will also return an error.
   
